openhands/events/serialization/action.py

```diff
@@ -84,6 +84,10 @@ def handle_action_deprecated_args(args: dict[str, Any]) -> dict[str, Any]:
                 'command'
             )  # "view" will be translated to FileReadAction which doesn't have a command argument
 
+    # Handle is_static field for CmdRunAction
+    if 'is_static' in args and args['is_static'] is None:
```

Comment:
```suggestion
    if 'is_static' in args:
```

@malhotra5 I think openhands-agent got the place right, though it thinks it's None for some reason.

I think we can just remove it from the dict. Give or take that I'm sleeping, so don't believe me, verify 😅

Reply:
Also, once this one passes, another attribute `cwd` needs the same.

Reply:
I actually think this PR may not be needed

Experimental resolver seems to be working

The released resolver isn't working because of a different issue that's unrelated to the runtime

Reply:
If you're sure? Isn't the is_static issue happening on `fix-me`?

Reply:
Ah, I see what you mean. You set it as `awaiting release`. OK

Reply:
Ah the static issue was happening on `fix-me-experimental` but turns out my workflow definition was incorrect

The `fix-me` label has `Token is invalid` error and the resolver doesn't even start

Reply:
Ahh hah, I see, quite a coincidence actually. On `fix-me` the `token is invalid` hides `is_static`, the execution hits the token issue earlier. 😅

`fix-me` runs nikolaik, it runs the image you saw on `experimental` - you should be right we get away with it _too_, once we release